django_hatstall/identities/templates/shdb/shdb_form.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'base.html' %}
+{% with active_page="about" %}
+
+{% block title %}Shdb params{% endblock %}
+
+{% block head %}
+    {{ block.super }}
+{% endblock %}
+
+{% block body %}
+<div class="container">
+  <h1> Parameters of the SortingHat database: </h1>
+  <br>
+  <form method="POST" action="" enctype="multipart/form-data">{% csrf_token %}
```

Comment:
Why using multipart/form-data? We are not sending files or other items that need multipart.